source/framework/vx_graph.c

```diff
@@ -700,6 +701,22 @@ VX_API_ENTRY vx_status VX_API_CALL vxSetGraphParameterByIndex(vx_graph graph, vx
             status = vxSetParameterByIndex(graph->parameters[index].node,
                                            graph->parameters[index].index,
                                            value);
+            vx_uint32 ref_index;
```

Comment:
Can you explain why this section of code was added?

Reply:
@lcweaver can you give us the ticket ID for the bug?

Reply:
@raphacan Looks like Steve sent an email on this as I was posting this--the bug ID is TIOVX-1700